### DISCUSSION TITLE
Merge queues availability for Github Team private repositories?

### DISCUSSION THREAD

**Discussion by artemisart** [community/community#131130] [Pull Requests] [answered]
### Select Topic Area

Question

### Body

Hello, are merge queues planned for private repositories outside of GitHub Enterprise Cloud? It's been "Generally Available" for one year now but we still can't use them in Team plan.
Thanks

**Comment by JohnImril** [comment, marked as answer]
Currently, merge queues are only available for GitHub Enterprise Cloud users and aren’t supported on the GitHub Team plan. GitHub hasn’t announced any plans to bring this feature to the Team plan yet, but they may expand it in the future based on demand. Keep an eye on GitHub’s updates or reach out to support to express your interest!

**Comment by raywonkari** [comment]
is it still the same?
